src/commands/game_commands/leaderboard.ts

```diff
@@ -35,14 +41,14 @@ export default class LeaderboardCommand implements BaseCommand {
     help = {
         name: "leaderboard",
         description: "View the KMQ leaderboard.",
-        usage: ",leaderboard\n,leaderboard [server | game] {page_number}\n,leaderboard page {page_number}\n,leaderboard [enroll | unenroll]",
+        usage: ",leaderboard {server | game} {page_number} {daily | weekly | monthly}\n,leaderboard [enroll | unenroll]",
```

Comment:
This usage is very confusing.. maybe split into another line 
`,leaderboard {page_number}`
`,leaderboard {server | game | daily | weekly | monthly} {page_number}`
`,leaderboard [enroll | uneroll]`